docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-SOAPCALL.xml

```diff
@@ -446,8 +446,16 @@
 END;
 ip := 'http://127.0.0.1:8022/';
 ips := 'https://127.0.0.1:8022/';
-ipspw := 'https://username:password@127.0.0.1:8022/';
 svc := 'MyModule.SomeService';
+ips_secret := 'secret:myConnectSecret';
+        /* assumes a secret named http-connect-myConnectSecret exists &amp; contains:
+        /*
+         {
+          "url": "https://127.0.0.1:8022/",
```

Comment:
@JamesDeFabia Sorry, one more thing.  You could use "service.example.com" for the url.  Any tiny part we can play toward people no longer using IPs directly is good.

Reply:
@afishbeck Good point! I made that change everywhere it was used in this file. Please look again